builtin/audit/syslog/backend.go

```diff
@@ -133,6 +153,9 @@ type Backend struct {
 	salt       *salt.Salt
 	saltConfig *salt.Config
 	saltView   logical.Storage
+
+	nodeIDList []eventlogger.NodeID
```

Comment:
Isn't this field redundant?  Can't we just fetch the keys out of nodeMap when registering the pipeline?

Reply:
It establishes the ordering of the nodes, which is important to the pipeline definition that will come later (where we actually use this field).

Reply:
In that case can we get rid of nodeMap and make this slice be a slice of nodes instead of IDs?

It's not a big deal, I'm not insisting, I just prefer (when it's easy) to not store the same data twice in two different ways.